examples/symlinks/summon-symlinks

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail;
```

Comment:
Can leave off the trailing `;` on this line and Lines 3, 10, and 14.

Reply:
done